llvm/test/MC/RISCV/long-conditional-jump-crash.s

```diff
@@ -0,0 +1,19 @@
+# RUN: llvm-mc %s -mc-relax-all -triple=riscv64 -filetype=obj \
```

Comment:
llvm/test/MC/RISCV/rv64-relax-all.s tests -mc-relax-all. Add the test there, perhaps rename the file?